src/containers/Chat/ChatMessages/ChatMessage/ChatMessageType/ChatMessageType.module.css

```diff
@@ -1,18 +1,24 @@
 .Image {
-  margin: 5px 0px;
-  max-width: 280px;
+  max-width: 284px;
   max-height: auto;
   cursor: pointer;
   background-size: cover !important;
+  text-align: center;
 }
 
 .Image img {
   overflow: hidden;
-  max-width: 200px;
+  max-width: 250px;
   height: auto;
   object-fit: contain;
+  text-align: center;
+}
+.video {
```

Comment:
Please follow uppercase naming convention for classes.

Reply:
yes. Updated these changes .